### ISSUE TITLE
needNewKFAfter not update

### ISSUE THREAD

**Issue by match08** [tum-vision/LDSO#49]
needNewKFAfter not update?, if not linearizeOperation must update, dso:
if(needKF) needNewKFAfter=fh->shell->trackingRef->id;

**Comment by NikolausDemmel** [comment]
Yes, looks like the flag `needNewKFAfter` is not really used since it is not updated, but I can't easily tell what are the consequences. Have you analysed in which scenario this causes an issue?

@gaoxiang12 did some refactoring compared to DSO, so the flag might also be obsolete.
